aladdin/commands/deploy.py

```diff
@@ -120,8 +100,16 @@ def deploy(
         }
         # Update with cluster rule values
         values.update(cr.values)
+        # Update with user-specified values file
+        values_files = [
+            f"--values={file_name}" for file_name in set_override_values
+            if "=" not in file_name
+        ]
         # Update with --set-override-values
-        value_overrides = {k: v for k, v in (value.split("=") for value in set_override_values)}
+        value_overrides = {
+            k: v for k, v in
+            (value.split("=") for value in set_override_values if "=" in value)
+        }
```

Comment:
I'm not sure I'm a big fan of mixing argument types like this. I can't imagine there's a realistic danger of a values file having an `=` character in it, but that's a risk with this current implementation.

I think the larger issue is that it seems fairly awkward to convey and utilize. Could we add a new argument like `--values-files`?